desk/app/groups.hoon

```diff
@@ -678,7 +679,18 @@
     (state-6-to-7 old)
   =?  cor  !=(~ caz-6-to-7)  (emil caz-6-to-7)
   =?  old  ?=(%7 -.old)  (state-7-to-8 old)
-  ?>  ?=(%8 -.old)
+  ::  v8 -> v9: leave all /contact subscriptions
```

Comment:
Seen in the wild: apparently some ships still have this subscription lingering, so we leave it for good during migration.